source/jormungandr/jormungandr/street_network/street_network.py

```diff
@@ -79,6 +86,10 @@ def make_path_key(self, mode, orig_uri, dest_uri, streetnetwork_path_type, perio
         """
         pass
 
+    @abc.abstractmethod
```

Comment:
I'd remove that abstract and have it return None by default (any connector who wants to add fp would overload, but not all)